src/reducers/columnStateHelper.js

```diff
@@ -71,9 +71,10 @@ function scrollTo(state, props, oldScrollToColumn, scrollX) {
   } = columnWidths(state);
   const fixedColumnsCount = fixedColumns.length;
 
-  const scrollToUnchanged = scrollToColumn === oldScrollToColumn
+  const noScrollableColumns = scrollableColumns.length === 0;
```

Comment:
349c257 [[diff](https://github.com/schrodinger/fixed-data-table-2/commit/349c25783ab6cd02018413174ea607ed46d2e6a6#diff-293ddd101833c33e0069803f5068f2f7R1207)]
Handles edge case when user tries to do a 'scroll to column' when no scrollable columns are present.